.mergify.yml

```diff
@@ -1,7 +1,7 @@
 pull_request_rules:
   - name: automatic merge on CI success and review
     conditions:
-      - status-success=continuous-integration/travis-ci/pr
+      - status-success=continuous-integration/taskcluster-ci/pr
```

Comment:
This should look like this: https://github.com/mozilla/code-review/blob/10967629839c02ef4a15db11c476fd49d5c2a445/.mergify.yml#L18.